.changeset/forty-colts-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'browserslist-config-seek': major
+---
+
+Update supported browsers
```

Comment:
I think it's worth mentioning the new supported browser versions here so it's easy to see in a renovate PR.

Reply:
Good idea